src/HttpHeaders.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace P4\MasterTheme;
+
+/**
+ * Class HttpHeaders
+ */
+class HttpHeaders {
+	/**
+	 * Headers constructor.
+	 */
+	public function __construct() {
+		add_action( 'wp_headers', [ $this, 'send_content_security_policy_header' ], 10, 1 );
+	}
+
+
+	/**
+	 * Send Content Security Policy (CSP) HTTP headers.
+	 */
+	public function send_content_security_policy_header() {
+		$allowed_frame_ancestors = [ '\'self\'' ];
```

Comment:
Would we add additional ancestors here, or will it always be `\self\`?

I guess there's also no reason to disallow `\self\', so maybe we shouldn't pass this to the filter, and can instead make the filter ask only for "extra" ancestors? That would in turn make the logic a bit simpler

Reply:
I agree, there's no reason to disallow `'self'`. I modified the filter so it only allows adding to the list of ancestors.